apps/api/src/__tests__/e2e_extract/index.test.ts

```
@@ -0,0 +1,340 @@
+              authors: { type: "array", items: { type: "string" } },
+            },
+          },
+        });
+
+      console.log(response.body);
+      expect(response.statusCode).toBe(200);
+      expect(response.body).toHaveProperty("data");
+      expect(response.body.data).toHaveProperty("authors");
+
+      let gotItRight = 0;
+      for (const author of response.body.data?.authors) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)